packages/update-monitor/src/types.ts

```diff
@@ -42,6 +43,12 @@ export interface ArbitrumProxyUpgradeability {
   userImplementation: string
 }
 
+export interface EIP1967ProxyUpgradeability {
+  type: 'eip1967 proxy'
+  adminSlot: string
+  implementationSlot: string
```

Comment:
I don't think these should be named "Slots", because they're the values of those slots, so "admin" and "implementation".